packages/neutrino-middleware-env/index.js

```diff
@@ -0,0 +1,5 @@
+const { EnvironmentPlugin } = require('webpack');
+
+module.exports = ({ config }, envs = []) => config
```

Comment:
Although it would be cool to map a name to preset/middleware options automatically, but that would probably require passing a name along with the preset/middleware. Which is possible.